libavfilter/dnn/dnn_backend_native.c

```diff
@@ -34,6 +34,7 @@
 #define FLAGS AV_OPT_FLAG_FILTERING_PARAM
 static const AVOption dnn_native_options[] = {
     { "conv2d_threads", "threads num for conv2d layer", OFFSET(options.conv2d_threads), AV_OPT_TYPE_INT,  { .i64 = 0 }, INT_MIN, INT_MAX, FLAGS },
+    { "async",          "use DNN async inference",      OFFSET(options.async),          AV_OPT_TYPE_BOOL, { .i64 = 1 },       0,       1, FLAGS },
```

Comment:
looks that the code in dnn_backend_native.c/h can be a separate patch, as the preparation for the async/sync unification of filter.

Reply:
like which part? The conversion to `TaskItem` based inference is in a separate patch.

Reply:
i meant the code in dnn_backend_native.c/h in the second patch. anyway, it doesn't matter to separate it.

Reply:
Okay